docs/reference/builder.md

```diff
@@ -1931,6 +2038,14 @@ required such as `zsh`, `csh`, `tcsh` and others.
 
 The `SHELL` feature was added in Docker 1.12.
 
+## Experimental features
```

Comment:
Probably needs a mention in the experimental docs; https://github.com/docker/cli/blob/master/experimental/README.md (ok for a follow-up)

Reply:
This is not the same experimental. Changed to external to avoid confusion.